tests/Feature/Http/Controllers/RatingControllerTest.php

```diff
@@ -35,12 +35,14 @@ public function setUp(): void
     public function index_displays_view()
     {
         $ratings = Rating::factory()->count(3)->create();
+        $user = User::factory()->has(Rating::factory()->count(3))->create();
+        $user2 = User::factory()->has(Rating::factory()->count(3))->create();
 
         $response = $this->get(route('rating.index'));
 
         $response->assertOk();
         $response->assertViewIs('rating.index');
-        $response->assertViewHas('ratings');
+        $response->assertViewHas('users');
```

Comment:
Your test case is a little under-specified, hence the low coverage, you can fix that by passing a couple of query parameters to the get request. the `route(...)` method natively support that for you so you can do:
`route('rating.index', ['name' => ..., 'search_langauge' => ..., etc])`

Reply:
thank you! I'll improve on the test coverage